timer in python

SeDe-cmd opened the pull request:
The timer uses the interface tkinter. With threading the application may use several more functions while the timer is running. This makes it possible to create more features such as pausing the timer or ingeneral having the application perform different tasks while the timer is running.

SeDe-cmd committed:
timer in python

The timer uses the interface tkinter. With threading the application may use several more functions while the timer is running. This makes it possible to create more features such as pausing the timer or ingeneral having the application perform different tasks while the timer is running.

GiovanniBruno22 commented on scripts/timer.py:
```diff
@@ -0,0 +1,54 @@
+
+timer_label = tk.Label(root, text="", font=("Times",48))
+
+def display_timer(seconds):
```
Trivial: Can we add docstrings to functions to document what happens and what they expect as input (if any) and return as output(if any)?
For example:
```
def display_timer(seconds):
    """ Display the timer.
         args: seconds (int)
    """
```

SeDe-cmd replied:
Is the new commits docstrings enough or do i need to go into more detail?

GiovanniBruno22 replied:
It is perfect, thank you so much for adding them!

SeDe-cmd committed:
Added docstrings to each function

The docstrings specifies what function does what and what sort of input and output one can expect from the functions

GiovanniBruno22 reviewed (approved):
Awesome work! thank you so much for this!
